src/calculateRentalCost.js

```diff
@@ -3,8 +3,31 @@
  *
  * @return {number}
  */
+
+const minDiscount = 20;
+const maxDiscount = 50;
+
+let discountSum = 0;
+const daysPrice = 40; /* $ */
+
 function calculateRentalCost(days) {
-  // write code here
+  const totalSum = days * daysPrice;
+
+  if (days < 3) {
+    return totalSum;
+  }
+
+  if (days === 3 || days < 7) {
```

Comment:
The condition `days === 3 || days < 7` is incorrect. It should be `days >= 3 && days < 7` to correctly apply the minimum discount for rentals between 3 and 6 days.